src/rpc/server.cpp

```diff
@@ -258,6 +258,22 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
     return "Bitcoin server stopping";
 }
 
+UniValue uptime(const JSONRPCRequest& jsonRequest)
+{
+    if (jsonRequest.fHelp || jsonRequest.params.size() > 1)
+        throw std::runtime_error(
+                "uptime\n"
+                        "\nReturns the total uptime of the Bitcoin server.\n"
+                        "\nResult:\n"
+                        "ttt        (numeric) The number of seconds that the Bitcoin server has been running\n"
```

Comment:
I used `ttt` because I saw it referenced in other commands for usage with time related values. For example in `getpeerinfo` you have `conntime` for a timestamp and also `timeoffset` for a time-based value in seconds. I think `ttt` will keep it consistent with other commands.